### ISSUE TITLE
A problem occurs when forcing the watch to force read HR. This was not happening in the JS Version

### ISSUE THREAD

**Issue by lunarlynix** [Furred/LynxHR#2]
Start Mesurement
```js
    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x02, 0x00]));
    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x01, 0x00]));

    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x01, 0x01]));
```

Not sure why it is not working in here.

**Comment by lunarlynix** [comment]
This would  provide 1s readings for the graphs in **NeosVR** or **VR Chat**

**Comment by lunarlynix** [comment]
Javascript Code
```js
console.log("Starting heart rate measurement")
    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x02, 0x00]));
    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x01, 0x00]));
    await this.startNotifications(this.chars.hrMeasure, (e) => {
      console.log("Received heart rate value: ", e.target.value);
      const heartRate = e.target.value.getInt16();
      window.dispatchEvent(
        new CustomEvent("heartrate", {
          detail: heartRate,
        })
      );
    });

    await this.chars.hrControl.writeValue(Uint8Array.from([0x15, 0x01, 0x01]));

    // Start pinging HRM
    this.hrmTimer =
      this.hrmTimer ||
      setInterval(() => {
        console.log("Pinging heart rate monitor");
        this.chars.hrControl.writeValue(Uint8Array.from([0x16]));
      }, 12000);
```

**Comment by lunarlynix** [comment]
We need to write ``[0x16]`` to **HR Control**

**Comment by lunarlynix** [comment]
Using this as a reference may solve it 
Source : https://github.com/AdrianDahlstrom/miband5/blob/master/miband.py

```python
 # stop heart monitor continues & manual
        char_ctrl.write(b'\x15\x02\x00', True)
        char_ctrl.write(b'\x15\x01\x00', True)
        # enable heart monitor notifications
        char_d.write(b'\x01\x00', True)
        # start hear monitor continues
        char_ctrl.write(b'\x15\x01\x01', True)
        t = time.time()
        while True:
            self.waitForNotifications(0.5)
            self._parse_queue()
            # send ping request every 12 sec
            if (time.time() - t) >= 12:
                char_ctrl.write(b'\x16', True)
                t = time.time()
```

**Comment by lunarlynix** [comment]
This issue is fixed as of the latest commit ``f680a0d86c05a111bfef8debb8ede216c1dcc66c``
